Engine/Core/ParserCommand.cpp

```diff
@@ -1554,7 +1554,7 @@ int ScriptParser::arcCommand() {
 		delete script_h.reader;
 		script_h.reader = new SarReader(archive_path);
 		if (script_h.reader->open(buf)) {
-			std::snprintf(script_h.errbuf, MAX_ERRBUF_LEN,
+			std::snprintf(script_h.errbuf, MIN_ERRBUF_LEN,
```

Comment:
This change looks wrong to me. script_h.errbuf is MAX_ERRBUF_LEN bytes.

Reply:
A linting error said it was a stack overflow risk

Reply:
I am afraid that linter is broken. Please revert.